plugins/codemirror/source.html

```diff
@@ -103,7 +103,8 @@
 
 	// Borrowed from codemirror.js themeChanged function. Sets the theme's class names to the html element.
 	// Without this, the background color outside of the codemirror wrapper element remains white.
-	document.documentElement.className += CMsettings.config.theme.replace(/(^|\s)\s*/g, " cm-s-");
```

Comment:
what you mean with check CMsettings.config.theme?
do we have to init that config in tinyMCE codemirror init part?